lib/commands/arm/appserviceplan/appserviceplan._js

```
@@ -226,24 +198,120 @@ exports.init = function (cli) {
+          skuInfo.capacity = numOfInstances;
+        } 
+        envelope.location = result.location;
+        try {
+          result = client.serverFarms.createOrUpdateServerFarm(resourceGroup, name, envelope, options , _);
+        } finally {
+          progress.end();
+        }
+        log.info('App Service Plan ' + name + ' has been updated ');
+      }
+    });
     
   function translateWorkerSize(skuName) {
     var workerSize = skuName.charAt(1);
     var workerSizeNum = Number(workerSize) - 1;
     var workerSizeLiteral = -1;
-	switch (workerSizeNum) {
-		case 0:
-      workerSizeLiteral = 'Small';
-      break;
-	  case 1:
-		  workerSizeLiteral = 'Medium';
-	    break;
-		case 2:
-		  workerSizeLiteral = 'Large';
-	    break;
-		default:
-	    workerSizeLiteral = 'Undefined';
+	  switch (workerSizeNum) {
+	  	case 0:
+        workerSizeLiteral = 'Small';
+        break;
+	    case 1:
+	  	  workerSizeLiteral = 'Medium';
+	      break;
+	  	case 2:
+	  	  workerSizeLiteral = 'Large';
+	      break;
+	  	default:
+	      workerSizeLiteral = 'Undefined';
 	  }
 	  return workerSizeLiteral;
 	}
+
+  function translateTier(tier) {
+    var tierUpper = tier.toUpperCase();
+    var workerSize = -1;
+    var sku = 'invalid';
+    switch(tierUpper) {
+      case 'FREE':
+      case 'F1':
+        tier = 'F1';
+        workerSize = 0;
+        break;
+      case 'SHARED':
+      case 'D1':
+        tier = 'D1';
+        workerSize = 0;
+        break;
+      case 'B1':
+      case 'B2':
+      case 'B3':
+        sku = 'Basic';
+        workerSize = tier.charAt(1) - 1;
+        break;
+      case 'S1':
+      case 'S2':
+      case 'S3':
+        sku = 'Standard';
+        workerSize = tier.charAt(1) - 1;
+        break;
+      case 'P1':
+      case 'P2':
+      case 'P3':
+        sku = 'Premium';
+        workerSize = tier.charAt(1) - 1;
+        break;
+      default:
+        throw new Error('Invalid Tier');
```

if you decide to change from `tier` to `sku`, then you should throw `invalid sku`